frontera/contrib/backends/remote/messagebus.py

```diff
@@ -78,3 +90,20 @@ def queue(self):
     @property
     def states(self):
         return None
+
+
+def aggregate_per_host(requests):
+    per_host = dict()
+    for link in requests:
+        if b'fingerprint' not in link.meta[b'domain']:
+            continue
+        host_fprint = link.meta[b'domain'][b'fingerprint']
```

Comment:
This is just a minor point, but maybe we could use the function defined below here. for e.g.

    for link in requests:
        host_fprint = get_host_fprint(link)
        if host_fprint is None:
            continue
        ...
The only issue I can think of with this code would be that it assigns a variable for all cases.